test/torchaudio_unittest/prototype/hifi_gan/original/meldataset.py

```diff
@@ -0,0 +1,45 @@
+# flake8: noqa
+# fmt: off
```

Comment:
Does fbcode linter recognize this?

Reply:
Alternatively, I think it's okay to format the code. It's simpler and no special treatment to carry on.